src/routes/safe/components/Settings/ManageOwners/index.tsx

```diff
@@ -28,173 +29,177 @@ import Img from 'src/components/layout/Img'
 import Paragraph from 'src/components/layout/Paragraph/index'
 import Row from 'src/components/layout/Row'
 import { getOwnersWithNameFromAddressBook } from 'src/logic/addressBook/utils'
+import { useAnalytics, SAFE_NAVIGATION_EVENT } from 'src/utils/googleAnalytics'
+import { AddressBookEntryProps } from 'src/logic/addressBook/model/addressBook'
+import { SafeOwner } from 'src/logic/safe/store/models/safe'
 
 export const RENAME_OWNER_BTN_TEST_ID = 'rename-owner-btn'
 export const REMOVE_OWNER_BTN_TEST_ID = 'remove-owner-btn'
 export const ADD_OWNER_BTN_TEST_ID = 'add-owner-btn'
 export const REPLACE_OWNER_BTN_TEST_ID = 'replace-owner-btn'
 export const OWNERS_ROW_TEST_ID = 'owners-row'
 
-class ManageOwners extends React.Component<any, any> {
-  constructor(props) {
-    super(props)
+const useStyles = makeStyles(styles)
 
-    this.state = {
-      selectedOwnerAddress: undefined,
-      selectedOwnerName: undefined,
-      showAddOwner: false,
-      showRemoveOwner: false,
-      showReplaceOwner: false,
-      showEditOwner: false,
-    }
-  }
+type Props = {
+  addressBook: unknown
```

Comment:
We have a type for addressBook

Reply:
I'm making use of it when it calls to `getOwnersWithNameFromAddressBook`, but If I define it at the Props level it fails. A refactor it's needed, but we should not take care of that in this PR.